packages/clay-autocomplete/src/Autocomplete.tsx

```diff
@@ -189,10 +196,12 @@ function hasItem<T extends Item>(
 const ESCAPE_REGEXP = /[.*+?^${}()|[\]\\]/g;
 
 const defaultMessages = {
+	allowsCustomValue: 'Add {0}',
```

Comment:
This should probably be called something more like "addNewItem" or "addCustomValue"